clinica/converters/aibl_to_bids/utils/json.py

```diff
@@ -0,0 +1,250 @@
+"""
+This module handles json writing from DICOM for PET modality in AIBL for BIDS 1.10 compliance
+"""
+
+import json
+from pathlib import Path
+from typing import Tuple, Union
+
+import numpy as np
+import pandas as pd
+from pydicom.dataset import DataElement, FileDataset
+from pydicom.multival import MultiValue
+from pydicom.tag import Tag
+
+from clinica.utils.stream import cprint
+
+# todo : __all__
+
+# todo : can this be used for other than AIBL ?
```

Comment:
Indeed, it's also what I'm wondering when reading all this.
We can keep it here and see later whether this could be useful for other converters, and move it if it turns out to be.